### PR TITLE
Member Request

### PR DESCRIPTION
**Pull request by shaunxw**
My earliest non-trivial PR:
- https://github.com/paritytech/substrate/pull/6237

Some notable PRs:
- https://github.com/paritytech/polkadot/pull/3538
- https://github.com/paritytech/cumulus/pull/510
- https://github.com/paritytech/substrate/pull/6763
- https://github.com/paritytech/polkadot/pull/2489

In addition to Polkadot/Substrate/Cumulus code contribution, I also
- contributed notably into `scale-info` back in 2019 https://github.com/paritytech/scale-info/commits?author=shaunxw
- created tools, for instance `xcm-emulator` to emulate and test XCM messages of production runtime. https://github.com/shaunxw/xcm-simulator#xcm-emulator
- reported bugs, such as UMP sink bug https://github.com/paritytech/polkadot/issues/3782, XCM encoding error https://github.com/paritytech/polkadot/issues/2490 etc.
- raised feature requests and so on.

I would like to apply for Rank 2.

### CODE REVIEW

**Comment by rphmeier** [comment]
I believe I or II Dan is more appropriate, 

> A member at this rank should be a core part of the team. Clear demonstration
of knowledge of the protocol is crucial, but since the individual knowledge of the
protocol is unlikely to be complete then general knowledge acquisition and communication/
sharing is also very important. The individual should be becoming
deeply familiar with at least one major area of the protocol in their role as a builder and should be becoming increasingly
familiar with design considerations typical in protocol development

Whereas the requirement of III Dan that

> becoming
a Fellow indicates the individual has enough knowledge and skill to build
substantial protocol components (i.e. all but the major protocol components,
e.g. a pallet or 2-10,000 line crate) alone and with high expectations that they
will be completed correctly and to a high standard. They should be able and willing to support that which they built
given that it is running 24/7 on a public network.

has not been demonstrated sufficiently by the links above

**Comment by gavofyork** [comment]
Indeed. I think the contributions back up a solid II Dan though, especially with the XCM simulator.

**Comment by shaunxw** [comment]
Thanks for the feedback. Updated my PR to apply for II Dan.

File: README.md [hunk 1]
```diff
@@ -14,4 +14,5 @@ The account/rank pairs which the Technical Committee should introduce.
 | `shawntabrizi` | `EGVQCe73TpFyAZx5uKfE1222XfkT3BSKozjgcqzLBnc5eYo` | 4 |
 | `kianenigma` | `HL8bEp8YicBdrUmJocCAWVLKUaR2dd1y6jnD934pbre3un1` | 4 | 
 | `ordian` | `GAWwkmjbLhM5pnAVbdZEkwd3QjHE7kaxJSCi3Ec91Q3QSDW` | 3 |
-| `KiChjang` | `GA3yPifemubFga7sTSFtLY2KFFiSRp6Bb8w31FS4xqgAvCz` | 2 |
\ No newline at end of file
+| `KiChjang` | `GA3yPifemubFga7sTSFtLY2KFFiSRp6Bb8w31FS4xqgAvCz` | 2 |
```
Comment (gavofyork) [inline README.md]: what's the actual change here??

**Review by gavofyork — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (shaunxw) [reply README.md]: ~It's probably my editor did some auto-formatting on saving the file. I'll try to revert it.~

It turns out to be `\ No newline at end of file`, which was moved from the line of `KiChjang` to the added new line.

The full diff log:

```
diff --git a/README.md b/README.md
index 3d456b7..71c7977 100644
--- a/README.md
+++ b/README.md
@@ -14,4 +14,5 @@ The account/rank pairs which the Technical Committee should introduce.
 | `shawntabrizi` | `EGVQCe73TpFyAZx5uKfE1222XfkT3BSKozjgcqzLBnc5eYo` | 4 |
 | `kianenigma` | `HL8bEp8YicBdrUmJocCAWVLKUaR2dd1y6jnD934pbre3un1` | 4 | 
 | `ordian` | `GAWwkmjbLhM5pnAVbdZEkwd3QjHE7kaxJSCi3Ec91Q3QSDW` | 3 |
-| `KiChjang` | `GA3yPifemubFga7sTSFtLY2KFFiSRp6Bb8w31FS4xqgAvCz` | 2 |
\ No newline at end of file
+| `KiChjang` | `GA3yPifemubFga7sTSFtLY2KFFiSRp6Bb8w31FS4xqgAvCz` | 2 |
+| `shaunxw` | `HxhDbS3grLurk1dhDgPiuDaRowHY1xHCU8Vu8on3fdg85tx` | 2 |
\ No newline at end of file
```

**Review by bkchr — APPROVED** [review]

**Review by jacogr — APPROVED** [review]

**Review by rphmeier — APPROVED** [review]

**Pull request merged**